e2e/404.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { test, expect, type Page } from '@playwright/test';
+import words from '../client/i18n/locales/english/motivation.json';
+
+let page: Page;
```

Comment:
```suggestion
import { test, expect } from '@playwright/test';
import words from '../client/i18n/locales/english/motivation.json';
```

We don't need a global `page` variable because...